dll/win32/kernel32/winnls/string/japanese.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         ReactOS system libraries
+ * FILE:            dll/win32/kernel32/winnls/string/japanese.h
+ * PURPOSE:         Japanese era support
+ * PROGRAMMER:      Katayama Hirofumi MZ
+ */
+
+#define JAPANESE_MAX_TWODIGITYEAR 99
+
+typedef struct JAPANESE_ERA
+{
+    WORD wYear;
+    WORD wMonth;
+    WORD wDay;
+    WCHAR szEraName[16];
+    WCHAR szEraInitial[5];
+    WCHAR szEnglishEraName[24];
+    WCHAR szEnglishEraInitial[5];
+} JAPANESE_ERA, *LPJAPANESE_ERA;
```

Comment:
I would drop the `L`, since all pointers are long these days...

Reply:
Dropped.